pulsar-broker/src/test/java/org/apache/pulsar/broker/admin/AdminApi2Test.java

```diff
@@ -147,6 +152,20 @@ protected ServiceConfiguration getDefaultConf() {
         return conf;
     }
 
+    @Override
+    protected MetadataStoreExtended createConfigurationMetadataStore() throws MetadataStoreException {
+        // use MockZooKeeperSession to provide a unique session id for each instance
+        return new ZKMetadataStore(MockZooKeeperSession.newInstance(mockZooKeeperGlobal), MetadataStoreConfig.builder()
```

Comment:
it shouldn't be better to make this the default behavior in `MockedPulsarServiceBaseTest ` ?

Reply:
Good idea, already changed